src/index.html

```diff
@@ -7,8 +7,42 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link 
+      rel="preconnect" 
+      href="https://fonts.gstatic.com" 
+      crossorigin="1">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="logo__box">
+        <img
+            src="images/logo.png"
+            alt="logo"
+            class="logo"
+          >
+      </a>
+      <nav class="nav">
+        <ul class="nav__panel">
+          <li class="list__item"><a href="#home" class="nav__link is-active" >Apple</a></li>
+          <li class="list__item"><a href="#home" class="nav__link">Samsung</a></li>
+          <li class="list__item"><a href="#home" class="nav__link">Smartphones</a></li>
```

Comment:
```suggestion
          <li class="list__item">
            <a href="#home" class="nav__link is-active" >Apple</a>
          </li>
          <li class="list__item">
            <a href="#home" class="nav__link">Samsung</a>
          </li>
          <li class="list__item">
            <a href="#home" class="nav__link">Smartphones</a>
          </li>
```
Please fix all cases

Reply:
Corrected